### PR TITLE
CDS-51349-HTTP Step supports API KEY in the auth header

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
HTTP Step supports API KEY in the auth header

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]
This looks good, thanks

File: docs/continuous-delivery/cd-technical-reference/cd-gen-ref-category/http-step.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ For example, in **Key**, enter `Token` .
 
 In **Value**, enter `<+secrets.getValue("aws-playground_AWS_secret_key")>`.
 
+You can copy the key and paste it in the HTTP step **Header** setting. See [Add and Manage API Keys](https://developer.harness.io/docs/platform/role-based-access-control/add-and-manage-api-keys/).
```
Comment (SudheendraKatte) [inline docs/continuous-delivery/cd-technical-reference/cd-gen-ref-category/http-step.md:70]:
```suggestion
You can copy the key and paste it in the HTTP step **Header** setting. For more information, go to [Add and Manage API Keys](https://developer.harness.io/docs/platform/role-based-access-control/add-and-manage-api-keys/).
```

**Review by SudheendraKatte — APPROVED** [review]
Approved with a minor comment.

**Review by SudheendraKatte — APPROVED** [review]
LGTM

**Pull request merged**